react/features/chat/types.ts

```diff
@@ -44,6 +45,11 @@ export interface IChatMessageProps extends WithTranslation {
      */
     canReply?: boolean;
 
+    /**
+     * Whether the chat message menu is visible or not.
+     */
+    chatMessageMenu?: boolean;
```

Comment:
Is this used anywhere? I remember it was renamed from kebabMenu but I think we no longer need it

Reply:
It's needed to determine wether to display the kebab menu only (on the left side, for one's own message) or both kebab menu and reactions menu for other participant's messages.